release.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -e
+
+# Usage
+# PUBLISH_GEM=true RELEASE_VERSION=1.2.7 ./release.sh
+
+PHRASEAPP_RUBY_REPO=git@github.com:phrase/phraseapp-ruby.git
+PHRASEAPP_RUBY_TMP=$(mktemp -d /tmp/phraseapp-ruby-lib-XXXX)
+
+
+function cleanup {
+  rm -Rf $PHRASEAPP_RUBY_TMP
+}
+
+function write_version {
+cat <<EOF > lib/phraseapp-ruby/version.rb
+module PhraseApp
+  VERSION = '${1}'
+end
+EOF
+}
+
+function push_to_git {
+  current_branch=$(git symbolic-ref --short HEAD)
+  if [[ "$current_branch" != "$1" ]]; then
+    git checkout -b $1
+  fi
+
+  git add .
```

Comment:
what about other local changes? would it be not enough to just both generated files?